<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelAreaDialog</class>
 <widget class="QDialog" name="ModelAreaDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>200</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Model Area</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="widthFactorLabel">
       <property name="text">
        <string>Width factor</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="widthFactorSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="heightFactorLabel">
       <property name="text">
        <string>Height factor</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="heightFactorSpinBox">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="topHorizontalLayout">
     <item>
      <widget class="QLabel" name="marginLabel">
       <property name="text">
        <string>Margin</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="marginHorizontalSlider">
       <property name="minimum">
        <number>75</number>
       </property>
       <property name="maximum">
        <number>175</number>
       </property>
       <property name="singleStep">
        <number>5</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>10</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="marginValueLabel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>75 px.</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="bottomHorizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>82</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="applyPushButton">
       <property name="text">
        <string>Apply</string>
       </property>
       <property name="icon">
        <iconset resource="../../rsc.qrc">
         <normaloff>:/icons/dialog-ok-apply.png</normaloff>:/icons/dialog-ok-apply.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
